api/src/main/java/io/onetable/client/CatalogConfig.java

```diff
@@ -15,7 +15,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
```

Comment:
nitpick: can we just keep this newline between the license and package?

Reply:
> nitpick: can we just keep this newline between the license and package?

@the-other-tim-brown Fixed, could you please help me take a look again? Thank you